main/next/plugins.yml

```diff
@@ -0,0 +1,127 @@
+# This is the index for all Tutor plugins that are officially maintained by Edly.
```

Comment:
You did do a `git mv`, right? Not sure of how GitHub displays those changes.

Reply:
This was `mv`, followed by `s/@nightly/@next`, followed by `git add .`.

Would you like me to break it into 2 commits so that each step is reviewable @regisb ?

Reply:
No, that's unnecessary. Thanks for the clarification!

Reply:
(This one is split into separate commits :)